.github/workflows/documentation.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Set up python
         uses: actions/setup-python@v1
         with:
-          python-version: '3.9'
+          python-version: 3.9
```

Comment:
I have changed this is in my pull request so that we use the latest version of the action (v4)